Sprint-1/fix/median.test.js

```diff
@@ -20,7 +15,6 @@ describe("calculateMedian", () => {
   test("doesn't modify the input", () => {
     const list = [1, 2, 3];
```

Comment:
If you expected the array to be sorted in the function, why not start with an array that is not sorted?